src/styles/index.scss

```diff
@@ -1,3 +1,245 @@
+:root {
+  --light-blue: #00acdc;
+  --column-count: 4;
+}
+
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+// #region header
+.header {
+  box-sizing: border-box;
+  padding-left: 50px;
+  padding-right: 50px;
+  width: 100%;
+  height: 60px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  justify-content: space-between;
+
+  &-logo {
+    display: block;
+  }
+}
+
+.nav {
+  &__list {
+    padding-inline-start: 0;
+    display: inline;
+  }
+
+  &__item {
+    display: inline-block;
+  }
+
+  &__link {
+    color: #060b35;
+    text-decoration: none;
+    text-transform: uppercase;
+    margin-left: 17px;
```

Comment:
You have a `margin-left: 17px;` property in your `.nav__link` class. This will add margin to the first link as well, which might not be what you want. Consider using `:not(:first-child)` selector to add margin only to the second and subsequent links.